src/cursor/cur_layered.c

```diff
@@ -1410,7 +1410,6 @@ __clayered_remove_int(
 
     if (S2C(session)->layered_table_manager.leader) {
         c = clayered->stable_cursor;
-        clayered->current_cursor = c;
```

Comment:
Is there a reason on why we move this to the end of the if statement? I understand the follower reason, but not sure if leader needs to follow the same reason